api/app/reviews/repository.py

```diff
@@ -86,6 +86,7 @@ def count_unassigned_reviews(event_id, required_reviews_per_response):
     def get_review_form(event_id):
         review_form = (
             db.session.query(ReviewForm)
+                    .filter_by(active=True)

```

Comment:
This comment all applies to all the repo methods updated with `active=True`. Is there going to be a case where an event organiser is going to want to pull review data from a previously deactivated stage?

Reply:
Great point @amritpurshotam - I didn't think about it, but you're totally right, an organiser will definitely want to pull data from previous stages, so in fact we should pass the stage to these repo methods rather than active=True. I'll make these changes before merging this PR.

Reply:
Will do this in a subsequent PR because I realised I have another branch building on top of this and don't want to deal with the merge conflicts 😂